rotkehlchen/types.py

```diff
@@ -353,6 +359,31 @@ def ens_coin_type(self) -> int:
             return 9000
         raise AssertionError(f'Invalid SupportedBlockchain value: {self}')
 
+    def get_evm_chain_id(self) -> int:
+        if self not in EVM_CHAIN_TO_ID:
+            raise DeserializationError('Failed to obtain id for chain')
+        return EVM_CHAIN_TO_ID[self]
+
+    @classmethod
+    def from_evm_chain_id(cls, chain_id: int) -> 'SupportedBlockchain':
+        enum_value = EVM_CHAIN_ID_TO_ENUM[chain_id]
+        if enum_value is None:
+            raise DeserializationError(f'Failed to obtain chain with id {chain_id}')
+        return enum_value
+
+
+EVM_CHAIN_TO_ID = {
+    SupportedBlockchain.ETHEREUM: 1,
+    SupportedBlockchain.OPTIMISM: 10,
+    SupportedBlockchain.BINANCE: 56,
+    SupportedBlockchain.GNOSIS: 100,
+    SupportedBlockchain.MATIC: 137,
+    SupportedBlockchain.FANTOM: 250,
+    SupportedBlockchain.ARBITRUM: 42161,
+    SupportedBlockchain.AVALANCHE: 43114,
+}
+EVM_CHAIN_ID_TO_ENUM = {value: key for key, value in EVM_CHAIN_TO_ID.items()}
```

Comment:
Left this is as a mapping here since having an Enum with duplicated keys felt incorrect